docs/sample_config.yaml

```diff
@@ -2068,6 +2068,21 @@ password_config:
       #
       #require_uppercase: true
 
+ui_auth:
+    # The number of milliseconds to allow a user-interactive authentication
+    # session to be active.
+    #
+    # This defaults to 0, meaning the user is queried for their credentials
+    # before every action, but this can be overridden to alow a single
+    # validation to be re-used.  This weakens the protections afforded by
+    # the user-interactive authentication process, by allowing for multiple
+    # (and potentially different) operations to use the same validation session.
+    #
+    # Uncomment below to allow for credential validation to last for 15
+    # seconds.
+    #
+    #session_timeout: 15000
```

Comment:
I debated if this should be separate for each operation? Something like:

```yaml
session_timeout:
  delete_device: 0
  reset_password: 0
```

It seemed like added complexity that I didn't want in the first version though. Thoughts?

Reply:
I'm minded to agree with you.